extra/1-radio-stations.js

```diff
@@ -14,6 +14,15 @@
  */
 
 // `getAllFrequencies` goes here
+function getAllFrequencies() {
+  let num = 86;
+  const arr = [];
+  for (let i = 0; i < 22; i++) {
+    num++;
+    arr.push(num);
+  }
+  return arr;
+}
```

Comment:
You can just use `i` here and remove `num` if you change the loop statement to ` for (let i = 86; i < 109; i++)`